src/modules/history.js

```diff
@@ -0,0 +1,54 @@
+/**
+* history
+* @author Deux Huit Huit
+*/
+(function ($, undefined) {
+
+	'use strict';
+
+	var scope = $('#site');
+
+	var sels = {
+		input: '.js-history-input'
+	};
+
+	var urls = [];
```

Comment:
Also dump it in the local storage ? Keep a lot of hard refreshes ?

Reply:
Good idea.

Reply:
1d12dabdea235aa236de3184a7cb7fd62a87dbd8